cartridges/app_storefront_training/cartridge/static/default/css/cart.css.map

```diff
@@ -0,0 +1 @@
+{"version":3,"sources":["webpack:///./cartridges/app_storefront_training/cartridge/client/default/scss/cartridges/app_storefront_training/cartridge/client/default/scss/cart.scss"],"names":[],"mappings":"AAAA;EACI","file":"default\\css\\cart.css","sourcesContent":[".hidden-promo {\r\n    display: none;\r\n}"],"sourceRoot":""}
```

Comment:
Don't commit the built files like .css and .css.map. When the branch reaches the development environment its own build is triggered